api-specs/test/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@commercetools-api-specs/test",
```

Is there going to be more than one "api-spec" package? Otherwise you can just make it a single package:

```suggestion
  "name": "@commercetools-local/api-specs",
```

---

I'm not sure what you mean but we have multiple api specs in commercetools.

---

You are adding here a package `test` under the `api-specs` folder. My question is if you plan to add other packages besides `test` under `api-specs` or not.

---

> You are adding here a package `test` under the `api-specs` folder. My question is if you plan to add other packages besides `test` under `api-specs` or not.

What is important to me is the possibility of adding other packages, so if I plan to or not does not matter to me. That is what I explained with my previous response.